publishable/database/seeds/MenuItemsTableSeeder.php

```diff
@@ -46,7 +44,7 @@ public function run()
                 'id'         => 3,
                 'menu_id'    => 2,
                 'title'      => 'Posts',
-                'url'        => "/{$prefix}/posts",
+                'url'        => route('posts.index'),
```

Comment:
Route names in Voyager should be prefixed with Voyager. This seems to be missing in many lines, please go though.